docker/lite/Dockerfile.ubi8.arm64.mysql80

```diff
@@ -0,0 +1,108 @@
+ && yum install -y --setopt=alwaysprompt=no --setopt=tsflags=nodocs --enablerepo c8base --enablerepo c8updates --enablerepo c8extras libev numactl-libs sysstat strace \
+ && yum install -y --setopt=alwaysprompt=no --setopt=tsflags=nodocs https://download-ib01.fedoraproject.org/pub/epel/8/Everything/aarch64/Packages/g/gperftools-libs-2.7-9.el8.aarch64.rpm https://download-ib01.fedoraproject.org/pub/epel/8/Everything/aarch64/Packages/j/jemalloc-5.2.1-2.el8.aarch64.rpm https://download-ib01.fedoraproject.org/pub/epel/8/Everything/aarch64/Packages/l/libunwind-1.3.1-3.el8.aarch64.rpm
+RUN yum update -y --setopt=alwaysprompt=no --setopt=tsflags=nodocs \
+ && yum install -y --setopt=alwaysprompt=no --setopt=tsflags=nodocs bzip2 ca-certificates gnupg libaio libcurl \
+    procps-ng rsync wget openssl hostname curl tzdata make \
+# Can't use alwaysprompt=no here, since we need to pick up deps
+#   No way to separate key imports and accept deps separately in yum/dnf
+ && yum install -y --setopt=tsflags=nodocs --enablerepo mysql80-community \
+    mysql-community-client mysql-community-server \
+# Have to use hacks to ignore conflicts on /etc/my.cnf install
+ && mkdir -p /tmp/1 \
+# Enable xtrabackup and percona toolkit repo using the new percona-release tool, commented out for now (no aarch64 packages)
+ #&& /usr/bin/percona-release enable-only pxb-80 \
+ #&& /usr/bin/percona-release enable pt \
+ #&& yum install -y --setopt=alwaysprompt=no --downloadonly --downloaddir=/tmp/1 --enablerepo mysql80-community percona-xtrabackup-80 percona-toolkit \
+ #&& rpm -Uvh --replacefiles /tmp/1/*rpm \
+ && rm -rf /tmp/1 \
+ && yum clean all \
+ && yum clean all --enablerepo mysql80-community \
+ && rm -rf /etc/my.cnf /var/lib/mysql /tmp/gpg /sbin/mysqld-debug
+
+# Set up Vitess user and directory tree.
+RUN groupadd -g 1001 -r vitess && useradd -r -u 1001 -g vitess vitess
+RUN mkdir -p /vt/vtdataroot && chown -R vitess:vitess /vt
+
+# Set up Vitess environment (just enough to run pre-built Go binaries)
+ENV VTROOT /vt/src/vitess.io/vitess
+ENV VTDATAROOT /vt/vtdataroot
+ENV PATH $VTROOT/bin:$PATH
+ENV MYSQL_FLAVOR MySQL80
+
+# Copy artifacts from builder layer.
+COPY --from=builder --chown=vitess:vitess /vt/install /vt
+COPY --from=builder --chown=vitess:vitess /vt/src/vitess.io/vitess/web/orchestrator /vt/web/orchestrator
+
+RUN mkdir -p /licenses
+COPY LICENSE /licenses
+
+# Create mount point for actual data (e.g. MySQL data dir)
+VOLUME /vt/vtdataroot
+USER vitess
+
+LABEL name="Vitess Lite image - MySQL Community Server 8.0" \
+      io.k8s.display-name="Vitess Lite image - MySQL Community Server 8.0" \
+      maintainer="cncf-vitess-maintainers@lists.cncf.io" \
+      vendor="CNCF" \
+      version="13.0.0" \
```

Comment:
This is the wrong version for main. I'd just remove it as I don't see where we have that in the other images. Or if we really need it, something like:
```
$ grep versionName go/vt/servenv/version.go | awk '{print $4}' | tr -d '"'
14.0.0-SNAPSHOT
```